axum/src/docs/routing/route.md

```diff
@@ -56,7 +56,22 @@ Note that `/*key` doesn't match empty segments. Thus:
 - `/*key` doesn't match `/` but does match `/a`, `/a/`, etc.
 - `/x/*key` doesn't match `/x` or `/x/` but does match `/x/a`, `/x/a/`, etc.
 
-Wildcard captures can also be extracted using [`Path`](crate::extract::Path).
+Wildcard captures can also be extracted using [`Path`](crate::extract::Path):
+
+```rust
+use axum::{
+    Router,
+    routing::get,
+    extract::Path,
+};
+
+let app = Router::new().route("/*key", get(handler));
```

Comment:
CI failure is actually legit, the compiler can't infer the type of `app` here (specifically the generic state type of the router).

```suggestion
let app: Router = Router::new().route("/*key", get(handler));
```